diff --git a/src/gtk/src/callbacks.c b/src/gtk/src/callbacks.c
index d2e99f6..8927f09 100644
--- a/src/gtk/src/callbacks.c
+++ b/src/gtk/src/callbacks.c
@@ -4610,9 +4610,6 @@ browse_url(signal_user_data_t *ud, const gchar *url)
     GtkWindow * parent;
     gboolean    result;
 
-    parent = GTK_WINDOW(GHB_WIDGET(ud->builder, "hb_window"));
-    result = gtk_show_uri_on_window(parent, url, GDK_CURRENT_TIME, NULL);
-    if (result) return;
     char *argv[] =
         {"xdg-open",NULL,NULL,NULL};
     argv[1] = (gchar*)url;
